src/DataCollection.UWP/LocalizedStrings/Resources.resw

```diff
@@ -519,4 +519,22 @@
   <data name="Of_ResultSeparator" xml:space="preserve">
     <value>of</value>
   </data>
+  <data name="AppInfo_About" xml:space="preserve">
+    <value>About</value>
+  </data>
+  <data name="AppInfo_LicenseButtonText" xml:space="preserve">
+    <value>View Licenses and Acknowledgements</value>
+  </data>
+  <data name="AppInfo_LicenseLink" xml:space="preserve">
+    <value>https://github.com/Esri/data-collection-dotnet#license</value>
+  </data>
```

Comment:
This value might be better served as a configuration stored in `Configuration.xml`. I'm not sure under what scenario a URL string will need to be altered based on the user's locale. It's also entirely possible I am misunderstanding the role of the LocalizedStrings Resources as they are used in this app...